src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -568,43 +570,80 @@ private void validateRules() {
       isValid = false;
     }
 
-    // If this is a multi-seat contest, we validate a couple extra parameters.
-    if (Utils.isInt(getNumberOfWinnersRaw()) && getNumberOfWinners() > 1) {
-      if (isSingleSeatContinueUntilTwoCandidatesRemainEnabled()) {
-        isValid = false;
-        Logger.log(
-            Level.SEVERE,
-            "winnerElectionMode can't be singleSeatContinueUntilTwoCandidatesRemain in a "
-                + "multi-seat contest!");
-      }
+    if (fieldOutOfRangeOrNotInteger(
+        getMultiSeatBottomsUpPercentageThresholdRaw(),
+        "multiSeatBottomsUpPercentageThreshold",
+        MIN_MULTI_SEAT_BOTTOMS_UP_PERCENTAGE_THRESHOLD,
+        MAX_MULTI_SEAT_BOTTOMS_UP_PERCENTAGE_THRESHOLD,
+        false)) {
+      isValid = false;
+    }
 
-      if (isBatchEliminationEnabled()) {
-        isValid = false;
-        Logger.log(Level.SEVERE, "batchElimination can't be true in a multi-seat contest!");
-      }
-    } else {
-      if (isMultiSeatSequentialWinnerTakesAllEnabled()) {
-        isValid = false;
-        Logger.log(
-            Level.SEVERE,
-            "winnerElectionMode can't be multiSeatSequentialWinnerTakesAll in a single-seat "
-                + "contest!");
-      } else if (isMultiSeatBottomsUpEnabled()) {
-        isValid = false;
-        Logger.log(
-            Level.SEVERE,
-            "winnerElectionMode can't be multiSeatBottomsUp in a single-seat contest!");
-      } else if (isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()) {
-        isValid = false;
-        Logger.log(
-            Level.SEVERE,
-            "winnerElectionMode can't be multiSeatAllowOnlyOneWinnerPerRound in a single-seat "
-                + "contest!");
-      }
+    boolean isMultiSeatBottomsUpPercentageThresholdSpecified =
+        getMultiSeatBottomsUpPercentageThresholdRaw() != null
+            && !getMultiSeatBottomsUpPercentageThresholdRaw().isBlank();
```

Comment:
Could you just set this to `getMultiSeatBottomsUpPercentageThreshold() != null`, or is the `.isBlank()` check important as well? If so, could you just put the latter in the former helper function and use it?